src/Mobs/PathFinder.cpp

```diff
@@ -0,0 +1,233 @@
+
+
+#include "Globals.h"
+#include "PathFinder.h"
+#include "../Chunk.h"
+
+cPathFinder::cPathFinder(double a_MobWidth, double a_MobHeight)
+	: m_Path(nullptr)
+	, m_GiveUpCounter(0)
```

Comment:
Consider comma use in constructor variable initialisation to be placed after variables, and colon use to be placed after the constructor definition as:

```cpp
cClass:cClass(...) :
   m_A(...),
   m_B(...),
   m_C(...)
```